### ISSUE TITLE
Support for multiple editions

### ISSUE THREAD

**Issue by sammys** [croneter/PlexKodiConnect#1984]
A similar issue was posted as #1867 however the response covered *versions* and not *editions*. Editions are displayed by Plex as separate entities rather than being a single entity that versions are. [Documentation by Plex](https://support.plex.tv/articles/multiple-editions/)

Using the default and Arctic Zephyr - Reloaded skins there doesn't appear to be any visible tagging of editions so I figured I ought to bring this up. Information of the entity does display the filename and that's what I've had to use to choose what to play.

Is there any existing support for displaying editions that I have overlooked? If not and displaying edition tags is possible somehow in Kodi I'm happy to scratch up a PR. If a PR is desired some advice/guidance is very welcome as I don't know the innards of Kodi yet!

**Comment by croneter** [comment]
Is there support for "editions" on the Kodi side of things, @sammys?

**Comment by sammys** [comment]
There are recent commits related to what looks like an overhaul of "versions". Haven't read any of the pull request threads nor code about it yet though.

**Comment by croneter** [comment]
Update: there is a Plex bug that prevents PKC from receiving a comprehensive list of all editions. This is necessary to implement editions. Plex is working on it

**Comment by sammys** [comment]
Great to hear. Thanks for the update!

I've run into an issue most likely caused by the new versions feature in Kodi Omega that doesn't appear to have been posted about yet. It also might be possible to address before the Plex bug is fixed.

I've tried the 3.8.4 beta with a new installation of Kodi Omega 21 beta 3 and it does a full initial sync of all Movies and TV Shows but the Kodi database does not end up containing any Movies yet does contain TV Shows. Pretty sure Kodi upgraded from pre-Omega will also not end up getting any Movies added to the Plex library after the upgrade.

Do you want me to post a new issue regarding this or leave it as part of this issue since the versions implementation in PKC will obviously cover this issue.

**Comment by croneter** [comment]
Use PKC beta 3.8.8 and you should be fine :wink:

**Comment by sammys** [comment]
A bucket load of commits flew in under the radar hehe. Thanks to both of you, @croneter and @Spacetech!

**Comment by sammys** [comment]
@croneter Tested a fresh installation of PKC beta 3.8.8 on Kodi 21 beta 3 on macOS and still no Movies shown after the initial sync.

**Comment by croneter** [comment]
Works for anyone else... Make tripple sure that you're using the latest PKC beta 3.8.8. Then reset the Kodi database again.

**Comment by sammys** [comment]
3.8.8 confirmed & database reset but PKC settings not reset. Still no Movies. Might be worth mentioning that I _did_ click on Ok when it told me it's going to restart Kodi. I waited around 30 seconds then exited and started it again.

Did a second round and, this time, also removed PKC settings. Waited 5 minutes and it still didn't restart itself though I didn't monitor it. Restarted, synced and no Movies.

Deleted the Kodi data directory in my home directory using the shell and reinstalled everything from scratch. That worked. Guess something in the Kodi data directory was borking everything.

**Comment by croneter** [comment]
Please open a new issue for that, @sammys, and share your entire DEBUG log file of the initial PKC sync. See https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by sammys** [comment]
@croneter If you're sure it's a valid use case I'll try to recreate it and post a new issue. It did arise from using a fresh installation of Kodi and a PKC version that doesn't have proper support for Omega (3.8.4) thus creating a "fictitious" database state. If you confirm you want it I'll use my development Plex library for it instead.

**Comment by artzox** [comment]
I recently moved from addon paths to direct ones and after repairing the DB I no longer get a context menu to open a specific version. You can see in plex that all versions are available, but when I check for versions in kodi these don't show.
I had to setup the plex server again as it disappeared on shield, versions were showing before that in pkc, with latest version and omega, it is only after I reinstalled plex and bound the server again that they disappeared.

**Comment by hipsofjw** [comment]
> Update: there is a Plex bug that prevents PKC from receiving a comprehensive list of all editions. This is necessary to implement editions. Plex is working on it

Has there been any update on this? Would be nice to be able to convert Plex "editions" into Kodi "versions".

**Comment by abacus-ear0r** [comment]
> > Update: there is a Plex bug that prevents PKC from receiving a comprehensive list of all editions. This is necessary to implement editions. Plex is working on it
> 
> Has there been any update on this? Would be nice to be able to convert Plex "editions" into Kodi "versions".

I'd be interested to know if there's been any change too. 

@croneter - thanks for the work you've put into PKC.

**Comment by mfrisina6600** [comment]
Thanks for the amazing work on PKC. I too would love to see editions support. Has there been an implementation since this was first created? On my end, when I have, say, a Theatrical edition and Director’s Cut edition of the same film, Kodi winds up showing the regular movie twice, with the only differentiating factor being the runtime between them.

**Comment by opnerppper** [comment]
Just commenting here real quick. As this is a huge hole missing IMO.
As i make allot of use of this plex feature. Allot of different versions just show up as standard in Kodi and PKC.
Any add-ons. Progress from Plex, or the PKC team on this?
